<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_hello">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>晚上好</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>75</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>游客</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>今日已学习</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hour">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>178</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>小时</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>总积分</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_score">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1023</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>个人学习</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>410</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>学习圈</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>470</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>用户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>恐龙邮差</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
